src/web/app/pages-instructor/instructor-session-base-page.component.ts

```diff
@@ -166,15 +167,11 @@ export abstract class InstructorSessionBasePageComponent {
    * Loads response rate of a feedback session.
    */
   loadResponseRate(model: SessionsTableRowModel): void {
-    model.isLoadingResponseRate = true;
-    const paramMap: { [key: string]: string } = {
-      courseid: model.feedbackSession.courseId,
-      fsname: model.feedbackSession.feedbackSessionName,
-    };
-    this.httpRequestService.get('/session/stats', paramMap).subscribe((resp: FeedbackSessionStats) => {
-      model.isLoadingResponseRate = false;
-      model.responseRate = `${resp.submittedTotal} / ${resp.expectedTotal}`;
-    }, (resp: ErrorMessageOutput) => { this.statusMessageService.showErrorMessage(resp.error.message); });
+    this.feedbackSessionsService.loadSessionStatistics(model)
```

Comment:
I don't think it is a good practice to pass the whole model into the service. The service should only get what are necessary.